source/sync/app-builder/local-to-sync.txt

```diff
@@ -86,30 +84,17 @@ language/platform:
 
 Copy Existing Data
 ------------------
+At this point, your data still only exists in the local database. You need to 
+perform an initial copy before Realm will sync the data. The Swift SDK and .NET 
+SDK provide an easy process for this step. For details, see:
```

Comment:
I might be wrong, but looking at the linked SDK pages, this is only applicable with PBS (not Flex Sync)? Do we want to make that more clear?

Reply:
Holy cow -- great catch! Confirming now if this is still limited to PBS or if we need to update those other docs.

Reply:
@cbullinger you were right. Updated.